doc/docs/faq.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Frequently Asked Questions
+slug: /faq
+---
+
+## How does the client know when to sync? Does it poll?
+
+Typically, servers send a *poke* (a content-less hint) over a websocket to tell the client to sync. There are many services you can use for this, and since no content flows over the socket there is no security/privacy concern.
+
+It is also possible to have Replicache poll using the [`pullInterval`](api/interfaces/replicacheoptions#pullinterval) field, but we only recommend this during development.
```

Comment:
I think it is important to still have a pullInterval in case the poke happens when the client is offline

Reply:
Good point, and for some applications polling is fine (Songbook does this). I'll adjust.